gdnative-core/src/nativescript/class.rs

```diff
@@ -452,6 +479,35 @@ where
     }
 }
 
+impl<'a, T, Access> RefInstance<'a, T, Access>
+where
+    Access: ThreadAccess,
+    T: NativeClass,
+    T::Base: GodotObject<RefKind = ManuallyManaged>,
+{
+    /// Returns `true` if the pointer currently points to a valid object of the correct type.
+    /// **This does NOT guarantee that it's safe to use this pointer.**
+    ///
+    /// # Safety
+    ///
+    /// This thread must have exclusive access to the object during the call.
+    #[inline]
+    #[allow(clippy::trivially_copy_pass_by_ref)]
+    pub unsafe fn is_instance_sane(&self) -> bool {
```

Comment:
This method is not necessary on `RefInstance` because it is always assumed to be valid.